src/main/java/hudson/plugins/sloccount/SloccountAreaRenderer.java

```diff
@@ -10,6 +11,7 @@
  * links. This renderer does not render tooltips, these need to be defined in
  * sub-classes.
  */
+@SuppressFBWarnings(value="EQ_DOESNT_OVERRIDE_EQUALS", justification="Equals method is not needed.")
```

Comment:
Honestly - it looks that new "maven" includes FB checks. I did not find any way to fix it (lots of other I fixed).
In old environment there was no complain for this piece of code.